tests/unit_tests/configuration/test_commandline_parser.py

```diff
@@ -61,3 +63,36 @@ def test_zenith_angle():
         match=r"The provided zenith angle, 190.0, is outside of the allowed \[0, 180\] interval",
     ):
         parser.CommandLineParser.zenith_angle(190)
+    with pytest.raises(ValueError):
+        parser.CommandLineParser.zenith_angle("North")
```

Comment:
See my other comment, I really like to give `North`.

Reply:
Well here it's irrelevant because for the zenith angle any string should fail

Reply:
I think I have to agree with that....